tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -246,9 +246,9 @@ def run(self):
                         except KeyError:
                             global VOLUME_TTL
                             # from second sync volume ttl is
-                            # SYNC_TTL + (no.volumes) * 30
+                            # SYNC_TTL + (no.volumes) * 30 + 200
```

Comment:
I worried when only one volume with large number bricks then  SYNC_TTL + (no.volumes) * 30 not enough. This change is only to handle one or two volume with a large number of brick in the same node.